chapter_CI_hypothesis/CI_and_hypothesis_test.Rmd

```diff
@@ -0,0 +1,1088 @@
+
+--
+
+* ***Interpretation***: the lower the p-value, the *less consistent our null hypothesis is with the observed statistic*.
+
+--
+
+* When do we decide to ***reject*** $H_0$ or not?
+
+---
+
+# Hypothesis Test Notation and Definitions
+
+* To decide wether we reject $H_0$ or not, we set a ***significance level*** for the test.
+
+--
+
+* ***Significance level $(\alpha)$:*** acts as a *cutoff* on the p-value.
+
+  * Common values are $\alpha = 0.01$, $0.05$, or $0.1$.
+
+--
+
+* ***Decision***:
+  * If the p-value falls ***below the cutoff $\alpha$***, we "***reject the null hypothesis at the significance level $\alpha$***."
+
+--
+
+  * Alternatively, if the p-value if ***greater than $\alpha$***, we say that we "***fail to reject the null hypothesis $H_0$ at the significance level $\alpha$***."
+
+--
+
+* ***Interpretation***: If what we observe *is too unlikely to happen* under the null hypothesis, it means that this hypothesis is ***likely to be false***.
+
+--
+
+* Let's illustrate how it works in our example. 
+
+---
+
+# Visualizing the P-value
+
+```{r,fig.height = 4.75, fig.width = 8, echo = FALSE}
+obs_diff_prop <- promotions %>%
+  specify(decision ~ gender, success = "promoted") %>%
+  calculate(stat = "diff in props", order = c("male", "female"))
+
+visualize(null_distribution, bins = 10, fill = "#d90502") + 
+  shade_p_value(obs_stat = obs_diff_prop,
+                size = 0.5,
+                direction = "right", 
+                fill = "black") +
+  labs(x = "Differences in sample proportions (male - female)",
+       y = "Frequency") +
+  xlim(-0.4, 0.4)+
+  theme_bw(base_size = 14) +
+  geom_vline(xintercept = 0.292, size = 1.25) +
+  annotate("text", x = 0.19, y = 485, label = "observed test statistic", size = 4)
+```
+
+--
+
+The shaded area correponds to the p-value!
+
+---
+
+# Obtaining the p-value and Deciding
+
+* Recall the definition of the p-value: ***probability of observing a test statistic *just as or more extreme* than the one we obtained, assuming the null hypothesis $H_0$ is true.***
+
+--
+
+* The shaded area corresponds to the share of ***test statistics under the null hypothesis*** that exceed the ***observed test statistic*** (0.292).
+
+```{r}
+p_value <- mean(null_distribution$stat >= 0.292)
+p_value
+```
+
+* In a world without discrimination, we would get $\hat{p_m} - \hat{p_f}$ superior (or equal) to 0.292 only `r 100*p_value`% of the time. 
+--
+
+* So, we can reject $H_0$, i.e. the absence of discrimination, at the 5% significance level.
+
+  * We also say that $\hat{p_m} - \hat{p_f} = 0.292$ is ***statistically significantly different from 0*** at the 5% level. 
+ 
+--
+
+* ***Question***: Suppose we had set $\alpha = 0.01 = 1\%$, would we have rejected the absence of discrimination at this level? 
```

Comment:
this kind of slide is too packed for my taste. there is too much stuff on it. I would try to put less or split them.

Reply:
i will take out the second line because it's really a repetition of the graph and we say it orally.